web/src/app/administration/import-export/import-export.component.ts

```diff
@@ -0,0 +1,61 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+import {ActivatedRoute, Router} from '@angular/router';
+import {WorkbasketService} from '../../services/workbasket/workbasket.service';
+import {ClassificationService} from '../../services/classification/classification.service';
+import {WorkbasketDefinitionService} from '../../services/workbasket/workbasketDefinition.service';
+
+@Component({
+  selector: 'taskana-import-export-component',
+  templateUrl: './import-export.component.html',
+  styleUrls: ['./import-export.component.scss']
+})
+export class ImportExportComponent implements OnInit {
+
+  @Input() currentSelection: string;
```

Comment:
I am using enum instead of strings, I prefer using enums since typescript is typed language, In javascript I would use strings.